changelog.d/18521_add_main_separator_in_kubernetes_logs.fix.md

```diff
@@ -0,0 +1,3 @@
+The `kubernetes_logs` source now handles parses log file paths on Windows to extract metadata.
+
+authors: @tmb-piXel
```

Comment:
```suggestion
authors: tmb-piXel
```

Actually we don't want the `@` :)

Reply:
Check Spelling didn't work without @ and I decided to add this symbol )

Reply:
Ah, gotcha, there is a change on `master` that avoids spellchecking the author bit of the changelog fragments due to false positives. I think that change just isn't in this branch yet.